### ISSUE TITLE
Fix positioning of content 

### ISSUE THREAD

**Issue by nicolocarpignoli** [kalwalt/jsartoolkit5#41]
Content is now shown at the left-bottom corner of the recognised image.
I think that we should try to make it appear at the center of the image, with the user able to move it, if wanted to - widespread use will be with the content at center of the image/covering the whole image.

**Comment by nicolocarpignoli** [comment]
@kalwalt you said on Slack, it's possible to understand width/height of given image. Where can we do that?

So knowing that, where we recognize the marker, we can try to change it position to:

position.x = position.x + width/2
position.y = position.y + height/2

**Comment by kalwalt** [comment]
When the NFT marker is loaded inside the loadNFTMarker:

https://github.com/artoolkitx/jsartoolkit5/blob/244b2b23286403e78fa24805b34509dc5a88052f/emscripten/ARToolKitJS.cpp#L313

the surfaceSet  pointer ( alias for a AR2SurfaceSetT struct) is filled with all the data from the NFT marker (width, height, dpi, image data, feature points...) so we can obtain all the data we needs from here.

AR2SurfaceSetT is a struct defined in https://github.com/artoolkitx/artoolkit5/blob/6742d9f5c4a06673a3805e097d924f8e0c55f8ca/include/AR2/tracking.h 

basically contain a chained series of structs inside of them there is an AR2ImageSetT struct where are stored the width, height dpi and image data:

https://github.com/artoolkitx/artoolkit5/blob/6742d9f5c4a06673a3805e097d924f8e0c55f8ca/include/AR2/imageSet.h#L52-L66

**Comment by nicolocarpignoli** [comment]
Great! Are you able to set two variables (width and height) that can then be available in the `artoolkit.api.js`? From there, we could just use them

**Comment by nicolocarpignoli** [comment]
Great! Are you able to set two variables (width and height) that can then be available in the `artoolkit.api.js`? From there, we could just use them

**Comment by kalwalt** [comment]
Yes, that should be doable ( in theory ): i will start to work to it.

**Comment by kalwalt** [comment]
Actually i solved in this way, given the **width**, **height** and **dpi** from the _.fset_ file we can calculate the position of the model in the center of the marker with:

```js
model.position.y = (msg.height / msg.dpi * 2.54 * 10)/2.0;
model.position.x = (msg.width / msg.dpi * 2.54 * 10)/2.0;
```

see this commit https://github.com/kalwalt/jsartoolkit5/pull/42/commits/86a19807000d9452ff537f8aa97775629c60d418

**Comment by evaristoc** [comment]
Hi, @kalwalt and @nicolocarpignoli,

I hope to help the procedure with some comments. I have been commenting the approach in the PR of this issue.  https://github.com/kalwalt/jsartoolkit5/pull/42#issuecomment-628906239
